src/NuGet.Clients/NuGet.VisualStudio.Implementation/Extensibility/VsPathContextProvider.cs

```diff
@@ -310,5 +312,63 @@ public IVsPathContext GetSolutionPathContext()
                 pathContext.FallbackPackageFolders.Cast<string>(),
                 trie);
         }
+
+        private IList<Project> GetProjectsInSolution(DTE dte)
+        {
+            var supportedProjects = dte.Solution.Projects;
```

Comment:
Remove this statement.  `supportedProjects` is unused.

Reply:
Good catch!